app/furniture/marketplace/README.md

```diff
@@ -78,7 +78,12 @@ You will need to use the [`stripe` cli](https://stripe.com/docs/stripe-cli) to f
 
 The `--forward-to` url can be found for a particular marketplace by using `polymorphic_url(marketplace.location(child: :stripe_events)` from within a controller or view debug session.
 
-You will then need to update the `marketplace#stripe_webhook_endpoint_secret` to match the webhook secret provided by the stripe cli.
+You will then need to update the `marketplace#stripe_webhook_endpoint_secret` to match the webhook secret provided by the stripe cli, via the Rails console or directly in the database.
+
+For the "Connect to Stripe" flow to work, you will need to have a local development URL that is recognized by Stripe as "reachable". One way to do this is:
+1. add `127.0.0.1 convene.local` to your `/etc/hosts`
+2. add `convene.local` as the `branded_domain` to whatever Space you are using for testing
```

Comment:
Technically, so long as you access the space via `convene.local` it should work; the `branded_domain` just shortens the URL to `convene.local:3000/rooms/:room_id/marketplaces/:marketplace_id` rather than `.../spaces/:space_id/...`

Reply:
Good point. Fixed.